### ISSUE TITLE
AD: dcinside.com

### ISSUE THREAD

**Issue by RoonMoonlight** [List-KR/List-KR#906]
### URL

Any posts in dcinside

### FILTERS

uBlock Filters (5/5)
EasyList
AdGuard mobile ads
EasyPrivacy
Online Malicious URL Blacklist
Peter Lowe's
EasyList/uBO Cookie notice
uBlock Filters - Annoyance
List-KR

### Browser

Mozilla Firefox

### Operating System

Android

### Ad-Blocker

uBlock Origin

### Comment

PowerLink also appears on any posts made from dcinside community. Sometimes search result also brings PowerLink as well.

![Screenshot_20240417_115906_Firefox](https://github.com/List-KR/List-KR/assets/25527589/550e8dc2-3364-4c46-918b-b989f5299fe1)

**Comment by piquark6046** [comment]
@RoonMoonlight Please update the filters list in your uBO. Then, please try again.

**Comment by RoonMoonlight** [comment]
> @RoonMoonlight Please update the filters list in your uBO. Then, please try again.

Same result

EDIT: Tried to clean the cookie and site data, same result.

**Comment by piquark6046** [comment]
Resolved.

**Comment by RoonMoonlight** [comment]
@piquark6046 PowerLink ads still displays regardless of updating the filter. I did add following to filter for workaround:
```
m.dcinside.com##section.clkhigh.grid:nth-of-type(4)
```

Thankfully this workaround solved the issue that continously displaying PowerLink ads.

**Comment by piquark6046** [comment]
@RoonMoonlight Is the powerlink ad not gone?

**Comment by RoonMoonlight** [comment]
> @RoonMoonlight Is the powerlink ad not gone?

Without temporary workaround, yes

https://m.dcinside.com/board/4defiant/662
This is the one of example which is affected by PowerLink (labelled as PowerLink Ads). It displays above list of post

**Comment by piquark6046** [comment]
Something was wrong in jsDelivr purging API.
I re-runed a job ( https://github.com/List-KR/List-KR/actions/runs/8717721373/job/23916713412#logs ).
Please update the filters lists and try again.

**Comment by RoonMoonlight** [comment]
> Something was wrong in jsDelivr purging API. I re-runed a job ( https://github.com/List-KR/List-KR/actions/runs/8717721373/job/23916713412#logs ). Please update the filters lists and try again.

Bad news: still the same result

**Comment by piquark6046** [comment]
Please inspect List-KR filters list in uBO and find `naverad`.
Does `||m.dcinside.com/ajax/*naverad` exists?

**Comment by RoonMoonlight** [comment]
> Please inspect List-KR filters list in uBO and find `naverad`. Does `||m.dcinside.com/ajax/*naverad` exists?

Nope, this one seems to be different than other PowerLink

![Screenshot_20240417_174103_Firefox](https://github.com/List-KR/List-KR/assets/25527589/3900612d-af46-448d-b0e7-0da6ce847b4b)

**Comment by piquark6046** [comment]
I meant List-KR filters view content button in your uBO configuration page.

**Comment by RoonMoonlight** [comment]
> I meant List-KR filters view content button in your uBO configuration page.

The filter just got updated to 2024.0417, same result

Yes there is that filter you mentioned there

**Comment by piquark6046** [comment]
So, Does the filtering not work well after your site data, cookies and cache is cleared?

**Comment by RoonMoonlight** [comment]
> So, Does the filtering not work well after your site data, cookies and cache is cleared?

Yes, still the same even I clear out the cookies and caches

**Comment by piquark6046** [comment]
Please tell me your ASN name after checking at https://ipinfo.io.

**Comment by RoonMoonlight** [comment]
> Please tell me your ASN name after checking at https://ipinfo.io.

I am using Mullvad VPN Osaka server (xtom)

**Comment by piquark6046** [comment]
It works well with Proton VPN JP#57 (xtom).
Can you access your computer to debug?

**Comment by RoonMoonlight** [comment]
> It works well with Proton VPN JP#57 (xtom). Can you access your computer to debug?

Why? I checked the PC version but there wasn't any PowerLink ads there

**Comment by piquark6046** [comment]
I meant https://firefox-source-docs.mozilla.org/devtools-user/about_colon_debugging/index.html#connecting-to-a-remote-device.

**Comment by piquark6046** [comment]
To see a request related to the advert on your side.

**Comment by RoonMoonlight** [comment]
> To see a request related to the advert on your side.

Okay all set up, what should I do?

**Comment by piquark6046** [comment]
Did you enter debugging tab of your Firefox mobile?
If so, please move to network tab and reload the webpage without cache.
Then, please find `naverad` keyword.

**Comment by RoonMoonlight** [comment]
Hmm okay seems it was solved automatically after some time... It says (random 7 character)naverad Blocked by uBlock Origin

I will let you know if anything else happens

**Comment by piquark6046** [comment]
> I will let you know if anything else happens

Please open new issue if so.
